XCUITests/SearchTest.swift

```diff
@@ -218,4 +218,38 @@ class SearchTests: BaseTestCase {
         waitForExistence(app.textFields["url"], timeout: 20)
         waitForValueContains(app.textFields["url"], value: "google")
     }
+    
+    func testSearchIconOnAboutHome() {
+        waitForTabsButton()
+        navigator.performAction(Action.OpenNewTabFromTabTray)
+        waitForTabsButton()
+        
+        // Search icon is displayed.
+        waitForExistence(app.buttons["TabToolbar.stopReloadButton"])
+        XCTAssertEqual(app.buttons["TabToolbar.stopReloadButton"].label, "Search")
+        XCTAssertTrue(app.buttons["TabToolbar.stopReloadButton"].exists)
+        app.buttons["TabToolbar.stopReloadButton"].tap()
+
+        let addressBar = app.textFields["address"]
+        XCTAssertTrue(addressBar.value(forKey: "hasKeyboardFocus") as? Bool ?? false)
```

Comment:
Not sure this is really working... I changed the Bool value to true and still working...